Add clarifications for squash commits & switching data sources (#23271)
DataDog/documentation: content/en/dora_metrics/setup/_index.md

{{< /whatsnext >}}

## Limitations

> - The first time you select a data source option (e.g. APM Deployment Tracking or PagerDuty), data is populated in DORA Metrics from that point in time.
> If you switch from source A to source B, then back to source A, historical data from source A will populate from the time when the source was first selected. 

Reviewer: ```suggestion
- When you first select a data source option (such as APM Deployment Tracking or PagerDuty), DORA Metrics begins populating data from that point forward. If you switch from source A to source B, then back to source A, the historical data from source A is only available from the time it was first selected. 
```
